code/controllers/configuration/configuration.dm

```diff
@@ -374,19 +375,39 @@
 	return runnable_modes
 
 /datum/controller/configuration/proc/LoadChatFilter()
-	GLOB.in_character_filter = list()
+	var/list/in_character_filter = list()
+	var/list/ooc_filter = list()
+
+	if(!fexists("[directory]/ooc_filter.txt"))
+		log_config("Error 404: ooc_filter.txt not found!")

```

Comment:
Please tell me this is just for the memes

Reply:
It is.